prisma/schema.prisma

```diff
@@ -38,3 +38,22 @@ enum FamilyStatus {
   DRAFT
   COMPLETE
 }
+
+model Project {
+  id String @id @default(cuid())
+
+  name        String
+  description String?
```

Comment:
I think it's better to change String to Text because description can be long.

Reply:
As Prisma [documentation](https://www.prisma.io/docs/concepts/database-connectors/postgresql#prisma-migrate) for text base datatype ( as string ), String was represented.
also this [docs](https://www.postgresql.org/docs/current/datatype-character.html)

Reply:
yes that's correct, but you need to specify @db.VarChar for name because name can have limited size.

Reply:
It's not better to do this validation on code instead of DB?

Reply:
if we use VarChar it will take less storage on database

Reply:
Please read the [docs](https://www.postgresql.org/docs/current/datatype-character.html) accurately.

> Don't specify a length maximum for database columns unless it is required by the application or you want to create a B-tree index on the column.
The performance will be the same, and PostgreSQL only allocates the space that is actually occupied by the data.

based on this [explaination](https://stackoverflow.com/questions/71396234/should-i-specify-a-size-to-a-postgresql-field-when-using-prisma)

Reply:
but we want B-tree index on id (default index type for primary key is B-Tree)